src/django_components/template_loader.py

```diff
@@ -2,33 +2,34 @@
 Template loader that loads templates from each Django app's "components" directory.
 """
 
-import os
 from pathlib import Path
 from typing import List, Set
 
-from django.apps import apps
 from django.conf import settings
 from django.template.loaders.filesystem import Loader as FilesystemLoader
 
 from django_components.logger import logger
 
 
-# Same as `Path.is_relative_to`, defined as standalone function because `Path.is_relative_to`
-# is marked for deprecation.
-def path_is_relative_to(child_path: str, parent_path: str) -> bool:
-    # If the relative path doesn't start with `..`, then child is descendant of parent
-    # See https://stackoverflow.com/a/7288073/9788634
-    rel_path = os.path.relpath(child_path, parent_path)
-    return not rel_path.startswith("..")
-
-
+# TODO - Document that `STATICFILES_DIRS` MUST have absolute paths
 class Loader(FilesystemLoader):
     def get_dirs(self) -> List[Path]:
+        """
+        Prepare directories that may contain component files:
+
+        Searches for dirs set in `STATICFILES_DIRS` settings. If none set, defaults to searching
+        for a "components" app. The dirs in `STATICFILES_DIRS` must be absolute paths.
+
+        Paths are accepted only if they resolve to a directory.
+        E.g. `/path/to/django_project/my_app/components/`.
+
+        If `STATICFILES_DIRS` is not set or empty, then `BASE_DIR` is required.
+        """
         # Allow to configure from settings which dirs should be checked for components
-        if hasattr(settings, "STATICFILES_DIRS") and len(settings.STATICFILES_DIRS):
+        if hasattr(settings, "STATICFILES_DIRS") and settings.STATICFILES_DIRS:
             component_dirs = settings.STATICFILES_DIRS
         else:
-            component_dirs = ["components"]
+            component_dirs = [settings.BASE_DIR / "components"]
```

Comment:
This has to be now full path instead of relative path, because also `settings.STATICFILES_DIRS` must be absolute paths.